### DISCUSSION TITLE
dnsmasq and global boot options

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#539] [Q&A] [answered]
Is it possible to have global boot options in dnsmasq (as a dhcp proxy) or you need to list all subnets in dnsmasq.conf?

**Comment by alkisg** [comment, marked as answer]
I had asked the dnsmasq developer about that ten years ago and he had said "all subnets are needed".

Maybe someone could ping him again (in the dnsmasq mailing list), in case it's possible to specify "any subnet" now. :)

**Reply by NiKiZe** [reply]
Last I checked there is a listen all option, however not really sure when and how that can be applied. So this is more of an "might exist" info.
